filters/shedder/admission.go

```diff
@@ -315,8 +315,8 @@ func (spec *AdmissionControlSpec) CreateFilter(args []interface{}) (filters.Filt
 		averageRpsFactor: averageRpsFactor,
 		totals:           make([]int64, windowSize),
 		success:          make([]int64, windowSize),
-		counter:          atomic.NewInt64(0),
-		successCounter:   atomic.NewInt64(0),
+		counter:          new(atomic.Int64),
+		successCounter:   new(atomic.Int64),
```

Comment:
Both should never be negative, so why do we need the change?

Reply:
This changes from uber to atomic libs

Reply:
That's what I already know :)

Reply:
Yes, that's why I changed them, because there was a linter error when comparison of different types was being done in the snippet you've given above. (uint64 vs int64).